angular-app/src/app/workspace/workspace.component.html

```diff
@@ -1,16 +1,20 @@
 <body>
-    <div id="workspace-content">
+    <div id="wcontent">
```

Comment:
I think workspace-content is actually a clearer name.

Reply:
I switched the id to help with formatting and I forgot to change it back